pkg/appmanager/appManager.go

```diff
@@ -1320,6 +1320,24 @@ func (appMgr *Manager) syncRoutes(
 						}
 					}
 				}
+				if dep.Kind == WhitelistDep {
```

Comment:
Since this is exactly the same as it is in syncIngress lets make this a common function between the two